modules/effects/src/concat_latest_from.ts

```diff
@@ -0,0 +1,38 @@
+import { Observable, of, OperatorFunction } from 'rxjs';
+import { concatMap, withLatestFrom } from 'rxjs/operators';
+
+type TypeOfObservable<T> = T extends Observable<infer U> ? U : never;
+
+/**
+ * 'concatLatestFrom' combines the source value
+ * and the last available value from a lazily evaluated Observable
+ * in a new array
+ */
+export function concatLatestFrom<T extends Observable<unknown>[], V>(
```

Comment:
@brandonroberts can you advise on what would be causing this to not render in the preview?

not only does it not show the comment, but the other overrides are not listed.

![Screen Shot 2020-12-29 at 20 25 23](https://user-images.githubusercontent.com/25187726/103324122-0f370f00-4a14-11eb-8f96-00ca8abd38e0.png)

Reply:
or @alex-okrushko, if you know what could be causing this, am looking for help

Reply:
@david-shortman, you might what to add the JSDoc comment just before the actual implementation (instead of the first overload).